### ISSUE TITLE
Topic: Create flow and phase flexibility idea for Phase II 

### ISSUE THREAD

**Issue by BeccaMelhuish** [citizenos/citizenos-fe#653]
@kevincrepin, We (@anettlinno @ilmartyrk @oksks and I) had a meeting recently where we discussed the flow of creating topic and also the order of the phases in the different topics, and we came up with an idea that we'd like to test out for a later phase (whether Phase II or later is tbc). We realise it's something that might need some thinking through, but this is our initial idea for you to play with (when there's time) and see what could work :)  

The idea is this: 

In the 'Create' menu, you would only have 'Topic' and 'Group'. 

When you choose 'Topic' you get taken to a preliminary tab/view, in which you choose which phase you would like to start your topic in, with copy along the lines of the below. 

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/3fff038f-c0d0-47d9-959f-0cd505649107)

You could also choose from a number of templates at this point, once we have templates ready, for different use cases and with different orders of phases already set in them. 

Once you're in the (non-template) topic, you can then decide which topics to add and in which order. You would have complete flexibility to add whichever phases you want, and in whichever order. 

For example, you could Vote > Discuss > Discuss > Ideate > Vote > Follow-up, if that's what your process needs. 

Our idea for the UX for adding phases is that this would happen here (though of course you may have other ideas!): 
![image](https://github.com/citizenos/citizenos-fe/assets/61320526/e99c2fc4-1c81-4033-a558-42792501dbff)

We'd also like to explore a kind of 'Follow-up' which doesn't have to be the final part of the process. 

For example, in one research call, a municipal government said their process for public consultations is this. So if they wanted to keep it all in one topic (which I understood they are keen to do), the process would be Discuss > Follow-up > Discuss > Follow-up. 

1. Initial ideas for development plan. 
2. Publish draft plan and action plan. 
3. Get feedback on the draft plan. 
4. Publish the final plan.

This idea links quite well with [this issue](https://github.com/citizenos/citizenos-fe/issues/251) too, as the intro to each phase would need to be different, so that the admin(s) can guide the participants through the different stages of the process. 

It will also need some thinking through in terms of what phases are open in parallel with each other too. E.g. you might sometimes want to keep one phase open in parallel with another phase, but you might sometimes want to close a phase once you've moved on to another phase.   

Once designed, we'd test out the prototype with users, to see if it makes sense for them, or just confuses things too much :)
